modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/CacheQueryType.java

```diff
@@ -43,5 +43,8 @@ public enum CacheQueryType {
     CONTINUOUS,
 
     /** SPI query. */
-    SPI
+    SPI,
+
+    /** Index query. */
+    INDEX
```

Comment:
Looks like now we need a test for new QueryType

Reply:
Test added, new functionality for recording index query events added.